aplib.net-demo/Assets/Scripts/Entities/Weapons/RangedWeapon.cs

```diff
@@ -5,14 +5,45 @@
 namespace Entities.Weapons
 {
     /// <summary>
-    /// This class has functions for ranged weapons.
+    /// This class has fuctions for ranged weapons.
```

Comment:
Why did you bring back the spelling mistake? Please make sure to merge things properly or ask for help if you are having issues, there have been multiple things seemingly brought back that were fixed on main after you created your branch :/